### PR TITLE
Shpakovski s eh1 t1

### PR DESCRIPTION
**Pull request by shpaser2**

### CODE REVIEW

**Review by AnzhelaKhamitskaya — APPROVED** [review]

**Review by gornovanton1992 — APPROVED** [review]
lgfm

**Review by DenisAbramovDm — APPROVED** [review]
Lgfm

File: homework/english/sergei_shpakovski/lecture1/task1.md [hunk 1]
```diff
@@ -1,3 +1,147 @@
-# Homework 1 task 1
+  
+GRAMMAR PRACTICE
+TASK1
+CHOOSE THE RIGHT OPTION (ВЫБЕРИТЕ ПРАВИЛЬНЫЙ ВАРИАНТ).
+1 He usually has breakfast at 6 a.m.
+2 Anyone who smokes 20 cigarettes a day can expect to have health problems.
+3 She doesn't like cooking, but today she bakes a pie.
```
Comment (Uchukaknado) [inline homework/english/sergei_shpakovski/lecture1/task1.md:7]: She doesn't like cooking, but today she's baking a pie.

File: homework/english/sergei_shpakovski/lecture1/task1.md [hunk 2]
```diff
@@ -1,3 +1,147 @@
...
+— She is a programmer.
 
-Hello world! This is my homework.
+TASK2
+CORRECT THE GRAMMAR MISTAKES (ИСПРАВЬТЕ ГРАММАТИЧЕСКИЕ ОШИБКИ)
+1 Do you usually work on Saturdays?
+2 Listen! Somebody is singing.
+3 Look! The sun is shining.
+4 He does not use his car every day.
+5 — Where did you come from?
```
Comment (Uchukaknado) [inline homework/english/sergei_shpakovski/lecture1/task1.md:19]: Where do you come from?

File: homework/english/sergei_shpakovski/lecture1/task1.md [hunk 3]
```diff
@@ -1,3 +1,147 @@
...
+1 to execute  -  выполнять
+2 to transfer - перемещать
+3 to compile - составлять
+4 to pass to - позволять, давать возможность
+5 to switch - перейти к
+6 to support - поддерживать
+7 to retool - переоборудовать
+8 to implement - применить
+9 to run a program - запустить программу
+10 to enable - применить
```
Comment (Uchukaknado) [inline homework/english/sergei_shpakovski/lecture1/task1.md:44]: позволять, давать возможность

File: homework/english/sergei_shpakovski/lecture1/task1.md [hunk 4]
```diff
@@ -1,3 +1,147 @@
...
+5 It's dark in your room. Are you sleeping?
+
+
+VOCABULARY PRACTICE
+TASK 1
+MATCH THE WORDS TO THEIR TRANSLATION (СОВМЕСТИТЕ СЛОВА ИЗ СПИСКА С РУССКИМ ПЕРЕВОДОМ)
+1 to execute  -  выполнять
+2 to transfer - перемещать
+3 to compile - составлять
+4 to pass to - позволять, давать возможность
```
Comment (Uchukaknado) [inline homework/english/sergei_shpakovski/lecture1/task1.md:38]: перейти к

File: homework/english/sergei_shpakovski/lecture1/task1.md [hunk 5]
```diff
@@ -1,3 +1,147 @@
...
+5 to switch - перейти к
```
Comment (Uchukaknado) [inline homework/english/sergei_shpakovski/lecture1/task1.md:39]: переключать

**Review by Uchukaknado — COMMENTED** [review]
Well done! Please, record the video/audio presentation and drop it in our chat.

**Pull request merged**